airflow/opt/providers/etna/etna/tests/conftest.py

```diff
@@ -118,8 +118,21 @@ def _get_client(self, auth):
     EtnaHook.get_client = _get_client
 
     # Authorization in any vcr is safely masked
-    return {"filter_headers": [("authorization", "XXX-Auth")]}
-
+    return {
+        "filter_headers": [("authorization", "XXX-Auth")],
+    }
+
+
+# When re-recording tests, you can use this callback in before_record_response
```

Comment:
That's fair.  I think I also ought to add additional encryption the way that our ruby tests currently do via `CI_SECRET`.  TODO